cmux-tui/CLAUDE.md

```
@@ -0,0 +1 @@
+Read and follow `AGENTS.md` in this directory before you change or verify `cmux-tui`.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)